### PR TITLE
Add statement on homepage about Lodash’s dependencies

### PR DESCRIPTION
**Pull request by ThibWeb**
**TL;DR;**

Another argument for "Why Lodash?" that I think is very relevant to today's ecosystem. While I don't think many people would expect a utility library to have dependencies, it is still worth stating.

---------------

The specific sentence I wrote isn't perfect. It overlooks the fact that some flavors of lodash do depend on other pieces of lodash – I think this is acceptable, since this is the same overall library it feels like an implementation detail. I'm not attached to the specific words either, if "zero dependencies" or "no external dependencies" sound better then 👍 . If the sentence would fit better elsewhere, 👍 .

I am proposing this change because I was surprised by the release of https://github.com/angus-c/just. There are a few things that this lib sells itself for that I think are very strong arguments for Lodash but are not documented enough:

- Lack of external dependencies (addressed by this PR)
- Package size – gzipped size of individual modular methods, and also the existence of the `babel` and `webpack` plugins (all of those things are mentioned in the docs, but without explaining why they matter).
- A third item would be performance – I think I know Lodash is fast, but all of the numbers I've seen have been from random blog posts and tweets rather than "official" info.

Now those 2 other items are a bit harder to address – just adding these here for the sake of completeness, but they should prob be discussed in separate issues.

### CODE REVIEW

**Comment by jdalton** [comment]
Thanks @ThibWeb!

I'm not super concerned with the project mentioned others like it.
They're a thin excuse for [NIH](https://en.wikipedia.org/wiki/Not_invented_here) and usually fade away in a few months. If users want to jump from a trusted collection of modules, like Lodash, to something else because of an arbitrary ideal dependency count then they're free to make poor choices.

**Comment by jdalton** [comment]
If I see projects like that start to gain traction I'll reconsider but so far folks on HackerNews, Reddit, and Twitter seem to get it.
